README.md

```diff
@@ -19,11 +19,15 @@ This client supports the following Google Cloud Platform services at a [Beta](#v
 * [Google BigQuery](#google-bigquery-beta) (Beta)
 * [Google Cloud Natural Language](#google-cloud-natural-language-beta) (Beta)
 * [Google Cloud Pub/Sub](#google-cloud-pubsub-beta) (Beta)
+* [Google Cloud Video Intelligence](#google-cloud-video-intelligence-beta) (Beta)
 * [Google Cloud Vision](#google-cloud-vision-beta) (Beta)
+* [Google DLP](#google-dlp-beta) (Beta)
+* [Google Stackdriver Error Reporting](#google-stackdriver-error-reporting-beta) (Beta)
+* [Google Stackdriver Monitoring](#google-stackdriver-monitoring-beta) (Beta)
 
 This client supports the following Google Cloud Platform services at an [Alpha](#versioning) quality level:
+* [Google Bigtable](#google-bigtable-alpha) (Alpha)
```

Comment:
Let's drop this until we are ready for the release

Reply:
thanks for the catch